client/src/components/BoardOfDirectors.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import './css/BoardOfDirectors.css';
+
+class BoardOfDirectors extends Component {
```

Comment:
Add comments to the top of these files

Reply:
What would the comments be for/of?

Reply:
just a small description of what the component's job is, where it's located on the frontend, and any things you deem necessary. Also any TODOs